src/components/AppRoutes.test.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { render, cleanup } from '@testing-library/react';
+import AppRoutes from './AppRoutes';
+import { withUserContext } from 'test/utils';
+import DashboardPage from 'views/dashboard/DashboardPage';
+import ConceptPage from 'views/concepts/ConceptPage';
+import ContactPage from 'views/contact/ContactPage';
+import ManageLocationPage from 'views/manage/ManageLocationPage';
+
+jest.mock('views/dashboard/DashboardPage');
+jest.mock('views/concepts/ConceptPage');
+jest.mock('views/contact/ContactPage');
+jest.mock('views/manage/ManageLocationPage');
+
+describe('AppRoutes', () => {
+  beforeEach(() => {
+    DashboardPage.mockReturnValue(<div>dashboard</div>);
+    ConceptPage.mockReturnValue(<div>concept</div>);
+    ContactPage.mockReturnValue(<div>contact</div>);
+    ManageLocationPage.mockReturnValue(<div>manage</div>);
+  });
+
+  afterEach(cleanup);
```

Comment:
cleanup is not needed with the latest version

Reply:
thanks!